### ISSUE TITLE
Android does not show user location

### ISSUE THREAD

**Issue by tombien** [nativescript-community/ui-mapbox#96]
After last update, I dont know why, but I am not able to show user location on Android. I did not make any changes from last version. I am stuck with that... Plese do you have any tips?

**Comment by tombien** [comment]
I find, that mayber bug is here
<img width="671" alt="Snímek obrazovky 2022-12-07 v 21 42 46" src="https://user-images.githubusercontent.com/19426771/206291441-d81d9850-390f-4a43-9e32-3bf868da3f8e.png">
 "showUserLocationMarker 2" is logged, but "showUserLocationMarker 3" not - I think this function does not end with resolve and is  interrupted here.

**Comment by farfromrefug** [comment]
@tombien can you try to add a try catch around that line and print the error?

**Comment by tombien** [comment]
@farfromrefug yea I can:
`Error: java.lang.IncompatibleClassChangeError: Found interface com.google.android.gms.location.FusedLocationProviderClient, but class was expected (declaration of 'com.google.android.gms.location.FusedLocationProviderClient' appears in /data/app/~~EbH2EsGKlyepoHXJnLM-kQ==/disway.trails-a6kNhxXFYIdoRJueoem9Jw==/base.apk!classes2.dex)`

**Comment by tombien** [comment]
Hello,
today I made as a workaround custom marker with blue dot icon. And I used NS Geolocation plugin and set a `watchLocation` to change marker coordinates.

**Comment by tombien** [comment]
@farfromrefug  yes, that is true, I will do it later. But I need to fix this very fast - so I made this with geolocation plugin for now.
Nobody from contributors does not know, where is the problem?

**Comment by guillemc23** [comment]
This happened to me too recently. I have my `showUserLocation` tag set to true, but when the map loads for the first time, asks for permissions and when approved, nothing happens, the map stays on its default coordinates and the user's blue dot never appears. Any workarounds?

**Comment by CatchABus** [comment]
@farfromrefug It seems this incompatibility issue has been fixed since mapbox v10.10.0 so plugin might need to use latest libraries.
<https://github.com/mapbox/mapbox-maps-android/issues/1752#issuecomment-1354535597>

**Comment by farfromrefug** [comment]
@CatchABus yes  but i wont update the plugin myself since it became closed source and requires special setup to add your key to simply download the sdk.
You can do it though but  for me i ll stop at the last open sourced version of mapbox ;)
I wont update that plugin anymore. Maybe i ll fork it one day if i need maplibre
